docs/developers/documentation/index.md

```diff
@@ -134,10 +162,21 @@ subtrees:
         - file: tutorials/geosciences/geo_tutorial2     # added
 ```
 
-## 4. Preview your document
+## 3. Preview your document
+
+If your documentation change includes code, it is important that you ensure
+the code is working and executable. This is why you will need to have a
+development installation of napari installed. [Examples](..../examples/README)
```

Comment:
I hope this makes sense. We just need them to check that their code works right? It is not otherwise executed as part of doc build?